ControlRoomApplication/ControlRoomApplication/GUI/InputDialog.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+
+/// <summary>
+/// Input dialog box class that can be used for any string input 
+/// </summary>
+namespace ControlRoomApplication.GUI
+{
+    public partial class InputDialog : Form
+    {
+        public InputDialog()
+        {
+            InitializeComponent();
+        }
+
+        public string getInput()
+        {
+            return textBox.Text;
+        }
+        public void setTitle(string title)
+        {
+            this.Text = title;
```

Comment:
We can actually set the title of the form from the control form without this method because `.Text` is public. So I believe this method can go as well.

- [ ] Consider removing `setTitle()`